autorest/azure/auth/authfile_test.go

```diff
@@ -0,0 +1,57 @@
+package auth
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestGetTokenFromAuthFile(t *testing.T) {
+	os.Setenv("AZURE_AUTH_LOCATION", getCredsPath())
+	auth, err := GetTokenFromAuthFile("https://management.azure.com")
+	if err != nil {
+		t.Fatalf("GetTokenFromAuthFile failed, got error %v", err)
```

Comment:
`Fatalf` should only be used when other errors wouldn't also make sense to report. For these first ones, it looks like that may make sense. Later in this Test though,it seems like you could replace this with subsequent calls to `t.Logf` and `t.Fail` respectively.

Reply:
Fixed